### PR TITLE
Update to 2020-01-14 release notes

### PR DESCRIPTION
**Pull request by thomasbroadwell**

### CODE REVIEW

File: Release Notes/2020/2020-01-14-cloud-platform-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+"date": "01-14-2020",
+"author": "Thomas Broadwell",
+"keywords":["centurylink", "release notes", "cam", "alm", "disconnect", "compute instances", "cloudformation"],
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### Announcements
+
+#### Cloud Application Manager Platform
```
Comment (gsanchezu) [inline Release Notes/2020/2020-01-14-cloud-platform-release-notes.md]:
```suggestion
#### [Cloud Application Manager Platform](https://www.ctl.io/cloud-application-manager/)
```

File: Release Notes/2020/2020-01-14-cloud-platform-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+##### Support for Portuguese and Spanish accents and special characters
+
+Cloud Application Manager now supports the usage of Portuguese and Spanish accents and special characters. Users can now name their resources with these types of characters, and also use them in most of the providers, boxes and instance attributes such as description, readme, claims and requirements, as well as in users' first and last names.
+
+#### Application Lifecycle Management
```
Comment (gsanchezu) [inline Release Notes/2020/2020-01-14-cloud-platform-release-notes.md]:
```suggestion
#### [Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management/)
```

File: Release Notes/2020/2020-01-14-cloud-platform-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Application Lifecycle Management has added the ability to bulk delete boxes in the boxes list view. Users can select several boxes in the list and if they have the necessary permissions to delete them, they can click on the Delete Boxes option in the Bulk Actions dropdown menu to remove them.
+
+##### Added support for Ubuntu 18 and Windows 2019 in CLC-based deployment policies
+
+Application Lifecycle Management now supports Ubuntu 18 and Windows 2019 OS-based images to be used in CLC deployment policies. Users should synchronize their CLC providers in order to gather information about the available images for these operating system versions to allow the selection of these as templates for their deployment policies.
+
+##### New CloudFormation types supported
+
+Application Lifecycle Management now supports additional CloudFormation types to be used in CloudFormation template boxes. These additional types are: "AWS::CodeBuild::ReportGroup" and "AWS::EC2::GatewayRouteTableAssociation". Users can now use these new resource types in the template definition of any CloudFormation template box or update any existing template instance to use it.
```
Comment (gsanchezu) [inline Release Notes/2020/2020-01-14-cloud-platform-release-notes.md]:
```suggestion
Application Lifecycle Management now supports additional CloudFormation types to be used in CloudFormation template boxes. These additional types are: "AWS::CodeBuild::ReportGroup" and "AWS::EC2::GatewayRouteTableAssociation". Users can now use these new resource types in the template definition of any CloudFormation template box or update any existing template instance to use them.
```

**Comment by thomasbroadwell** [comment]
Trying again to get this to be mergable

**Pull request closed without merging**